index.html

```diff
@@ -15,20 +15,25 @@
         <div class="container">
           <nav class="navbar flex">
             <div class="logo">
-              <a href="index.html"><h2>My Logo</h2></a>
+              <h2>My Logo</h2>
             </div>
             
-            <div class='menu-list'>
-              <ul>
-                <li><a href="#work-station" aria-label ="portfolio area"></a>Portfolio</li>
-                <li><a href="#about-station" aria-label ="about me area"></a>About</li>
-                <li><a href="#contact-station" aria-label ="contact me area"></a>Contact</li>
-              </ul>
-            </div>
-
+            <ul class="navnav-menu">
+              <li>
+                <a href="#work-section" class="navnav-items" style="text-decoration: none;">Portfolio</a>
+              </li>
+              <li>
+                <a href="#about-section" class="navnav-item" style="text-decoration: none;">About</a>
+              </li>
+              <li>
+                <a href="#contact-section" class="navnav-item" style="text-decoration: none;">Contact</a>
+              </li>
+            </ul>
 
-            <div class="menu-bar">
-              <img src="./images/menu-bar.png" alt="menu-bar" />
+            <div class="hamburger">
+              <span class="barbar"></span>
+              <span class="barbar"></span>
+              <span class="barbar"></span>
             </div>
           </nav>
```

Comment:
The styling for the menu is currently not consistent with the Figma design. Please fix the space issue (between the close icon and the nav links and between the top of the blue box and the nav links) and colour for the close button in order to be consistent with the Figma design.

YOUR DESIGN

![navSpace](https://user-images.githubusercontent.com/84930355/223440935-a2bee6d2-818a-44fd-b6d7-5ff603f97d53.PNG)


FIGMA

![betterNavSpace](https://user-images.githubusercontent.com/84930355/223440986-5b91fa3c-572d-4b95-8f3b-3f6348f51273.PNG)